Polyfills/Window/Source/Windows/WindowPlatform.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <Windows.h>
+#include <Babylon/JsRuntime.h>
+
+#include "..\Window.h"
+
+using WindowType = HWND;
+
+namespace Babylon::Polyfills::Internal
+{
+    Napi::Value Window::GetDevicePixelRatio(const Napi::CallbackInfo& info)
```

Comment:
Why is this in a header file instead of a cpp file? I feel like what I would have expected is:

```
Window\Source\Window.h
Window\Source\Window.cpp
Window\Source\Windows\Window.cpp
Window\Source\iOS\Window.cpp
Window\Source\Android\Window.cpp
etc.
```

But I'm not sure if there are different existing patterns in Babylon Native for this.

Reply:
I originally modeled this platform specific implementation off of:
https://github.com/BabylonJS/BabylonNative/blob/master/Dependencies/xr/Source/OpenXR/Windows/XrPlatform.h

But if that's not a great pattern, I'm definitely open to change it to match your pattern as well.

Reply:
Also according to @CedricGuillemet's comment above:https://github.com/BabylonJS/BabylonNative/pull/582#discussion_r562749856


We have 3 different patterns, It may be a good idea to align to a single pattern here.

Reply:
id personally prefer to cpp and header files for this one

---
In reply to: [563957901](https://github.com/BabylonJS/BabylonNative/pull/582#discussion_r563957901) [](ancestors = 563957901)

Reply:
> We have 3 different patterns, It may be a good idea to align to a single pattern here.

I only see two patterns. The difference between switching the header vs. the cpp just depends on where the code needs to change. This decision needs to be made regardless of the platform switching.

I am personally fine with both patterns. If it's a simple switch there is no need to create a directory to hold it. If there are lots of files, then a directory makes more sense.

Reply:
Ok, moved the implementation to a .cpp, following the (Source or Include)/${BABYLON_NATIVE_PLATFORM}/File convention